src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -114,29 +133,62 @@ module.exports = function<T, P, I>(config : HostConfig<T, P, I>) : Reconciler {
       // progress.
       const current = workInProgress.alternate;
       const next = completeWork(current, workInProgress);
+
+      // The work is now done. We don't need this anymore. This flags
+      // to the system not to redo any work here.
+      workInProgress.pendingProps = null;
```

Comment:
For most tags, can we throw away pendingProps in BeginWork? Is that better for the GC?